src/ontodia/data/sparql/sparqlDataProvider.ts

```diff
@@ -313,16 +357,20 @@ export function executeSparqlQuery<Binding>(endpoint: string, query: string, met
             return response.json();
         } else {
             const error = new Error(response.statusText);
-            (<any>error).response = response;
+            (<any> error).response = response;
             throw error;
         }
     });
 };
 
-export function executeSparqlConstruct(endpoint: string, query: string, method: SparqlQueryMethod): Promise<Triple[]> {
+export function executeSparqlConstruct(
+    endpoint: string,
+    query: string,
+    method: SparqlQueryMethod,
+    queryFunction: QueryFunction): Promise<Triple[]> {
```

Comment:
Move closing parenthesis to the next line (there should be a linting warning but I'm not sure that it detects this right now).